ui/src/components/credentials/IssuanceMethodForm.tsx

```diff
@@ -52,11 +52,16 @@ export function IssuanceMethodForm({
 
   const fetchConnections = useCallback(
     async (signal: AbortSignal) => {
-      const response = await getConnections({ credentials: false, env, signal });
+      const response = await getConnections({
+        credentials: false,
+        env,
+        params: { max_results: 0 },
```

Comment:
Why this `max_results: 0`?

Reply:
Removing it. Not needed. Probably I added it when debugging

Reply:
My intention was to download all connections from server to be shown in a  select. But, I think that this select is now an input text, so I think that this function has no sense right now. And can be a performance problem if the number of connections is too big.

Is this screen
![imagen](https://github.com/0xPolygonID/issuer-node/assets/15073263/fe371f5f-187f-4cb9-b2d5-e590915edc8f)

Reply:
As [discussed in Slack](https://0xpolygon.slack.com/archives/C05AG4LR7ML/p1705679168364929?thread_ts=1705655683.487219&cid=C05AG4LR7ML) there's a bug in the API that returns `0` in `max_results`, making the response invalid, which means that this dropdown cannot be populated.

As agreed, we will set this page size (`max_results`) to `1000` and will sort the connection `DESC` so users will see the most recent `1000` connections listed, which we think covers most use cases.

Reply:
Error fixed. Now server returns 1000 rows if parameter maxRows is not present.